src/components/Editor/Runners/PythonRunner/OutputViewToggle.js

```diff
@@ -25,31 +30,30 @@ const OutputViewToggle = () => {
   };
 
   return (
-    <div
-      className={`output-view-toggle`}
-      disabled={codeRunTriggered || drawTriggered}
-    >
-      <Button
-        className={`btn--small output-view-toggle__button output-view-toggle__button--tabbed${
-          isSplitView ? "" : " output-view-toggle__button--active"
-        }`}
-        buttonOuter
-        disabled={codeRunTriggered || drawTriggered}
-        label={t("outputViewToggle.buttonTabLabel")}
-        title={t("outputViewToggle.buttonTabTitle")}
-        ButtonIcon={TabbedViewIcon}
-        onClickHandler={switchToTabbedView}
-      />
+    <div className="output-view-toggle">
       <Button
-        className={`btn--small output-view-toggle__button output-view-toggle__button--split${
-          isSplitView ? " output-view-toggle__button--active" : ""
-        }`}
-        buttonOuter
+        className={"btn--tertiary output-view-toggle__button"}
+        buttonText={
+          isMobile
+            ? null
+            : isSplitView
+            ? t("outputViewToggle.buttonTabLabel")
+            : t("outputViewToggle.buttonSplitLabel")
+        }
```

Comment:
💭 I'm not normally a fan of nested tertiary operators but I get why you chose this over the spread syntax 😅 it's a little hard to read this huh, I think I prefer it but it's definitely down to personal preference this one 👍 
```
{...(!isMobile && {
  buttonText: isSplitView
    ? t("outputViewToggle.buttonTabLabel")
    : t("outputViewToggle.buttonSplitLabel"),
})}
```